_includes/pbc/all/install-features/202400.0/install-the-service-points-cart-checkout-feature.md

```diff
@@ -0,0 +1,142 @@
+{
+    /**
+     * @return list<\SprykerShop\Yves\CheckoutPageExtension\Dependency\Plugin\CheckoutAddressStepPostExecutePluginInterface>
+     */
+    protected function getCheckoutAddressStepPostExecutePlugins(): array
+    {
+        return [
+            new ServicePointCheckoutAddressStepPostExecutePlugin(),
+        ];
+    }
+}
+```
+
+2. Enable the demo Click & Collect replacement strategy plugins:
+
+For the demo purpose, we propose the example of the Click & Collect replacement strategy.
+
+| PLUGIN                                                                   | SPECIFICATION                                                                                                        | PREREQUISITES | NAMESPACE                                                                |
+|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|---------------|--------------------------------------------------------------------------|
+| ClickAndCollectExampleDeliveryServicePointQuoteItemReplaceStrategyPlugin | Replaces product offers in quote items that have `delivery` shipment type with suitable product offers replacements. |               | Spryker\Zed\ClickAndCollectExample\Communication\Plugin\ServicePointCart |
+| ClickAndCollectExamplePickupServicePointQuoteItemReplaceStrategyPlugin   | Replaces product offers in quote items that have `pickup` shipment type with suitable product offers replacements.   |               | Spryker\Zed\ClickAndCollectExample\Communication\Plugin\ServicePointCart |
+
+**src/Pyz/Zed/ServicePointCart/ServicePointCartDependencyProvider.php**
+
+```php
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Pyz\Zed\ServicePointCart;
+
+use Spryker\Zed\ClickAndCollectExample\Communication\Plugin\ServicePointCart\ClickAndCollectExampleDeliveryServicePointQuoteItemReplaceStrategyPlugin;
+use Spryker\Zed\ClickAndCollectExample\Communication\Plugin\ServicePointCart\ClickAndCollectExamplePickupServicePointQuoteItemReplaceStrategyPlugin;
+use Spryker\Zed\ServicePointCart\ServicePointCartDependencyProvider as SprykerServicePointCartDependencyProvider;
+
+class ServicePointCartDependencyProvider extends SprykerServicePointCartDependencyProvider
+{
+    /**
+     * @return list<\Spryker\Zed\ServicePointCartExtension\Dependency\Plugin\ServicePointQuoteItemReplaceStrategyPluginInterface>
+     */
+    protected function getServicePointQuoteItemReplaceStrategyPlugins(): array
+    {
+        return [
+            new ClickAndCollectExampleDeliveryServicePointQuoteItemReplaceStrategyPlugin(),
+            new ClickAndCollectExamplePickupServicePointQuoteItemReplaceStrategyPlugin(),
+        ];
+    }
+}
+```
```

Comment:
* We have to add the "Verification" section and describe a simple flow that covers at least one of the strategies.